src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -392,78 +237,79 @@ def static_dag(job,
     # - a spark driver
     # - a spark master/hdfs namenode
     # - _n_ spark workers/hdfs datanodes
-    if (pipeline_to_run == "adam" or
-        pipeline_to_run == "both"):
-        if adam_inputs['masterIP']:
+    if (inputs.pipeline_to_run == "adam" or
+        inputs.pipeline_to_run == "both"):
+        if inputs.master_ip:
             nodes_needed_after_alignment += 1
         else:
-            nodes_needed_after_alignment += (adam_inputs['numWorkers'] + 2)
+            nodes_needed_after_alignment += (inputs.num_nodes + 1)
```

Comment:
Yes, `num_nodes` is the total number of nodes in the toil spark cluster rather than the number of workers. This was changed for consistency with the ADAM pipeline.